docs/conf/portland/2018/news/volunteers-shirts-tickets.rst

```diff
@@ -0,0 +1,66 @@
+The campaign will run until **April 15th** to allow for timely delivery of your shirts ahead of the conference.
+
+And don't worry, you'll still have a Write the Docs Portland 2018 souvenier from the conference even if you decide not to order a shirt.
+We'll send another update once we finalized the design and product.
+
+Call for volunteers
+-------------------
+
+There is plenty to do during the conference and volunteers are essential to the success of the conference. We are looking for folks who are willing to help out during the conference. If you volunteer for two or more shifts (clocking in at an average of three hours each), you will receive free registration for the conference.
+
+We are looking for folks to help out in these roles:
+
+- Ballroom setup - Prepping the main conference hall for the main conference, including laying out cables/power strips, arranging tables/chairs, and other light physical labor
+- Writing day - Registering people and general information inquiries, most likely directions to various rooms and schedule questions, assist with other duties as assigned
+- Registration/Info - Registering people in the main conference hall and answering any questions
+- Speakers - Finding speakers and making sure that they're where they need to be when it comes time to speak
+- Talk Timing - Keeping track of the length of the talks and warning speakers when they're running out of time
+- Floaters - Other duties as assigned ;) Assisting with odds and ends requests from other volunteers and organizers as well as answering questions; overall, being generally helpful
+- A/V Troubleshooting - This on-call role will assist presenters at the time of their greatest need while on stage - issues with slides, video, or audio portions of their presentation. Please apply for this role if you have some A/V knowledge, experience troubleshooting different operating systems is a plus!
+- Unconference - Camping out in the unconference area downstairs, keeping the place in order, updating unconference room schedule, and so on
+- Job Fair setup/take down- Helping vendors set up their tables before the job fair, or helping with setting the room back for the preconferences that follow in the afternoon
+- Ballroom take down - What’s more fun than setting things up? Taking them down! Helping with taking down the conference set up and leaving the conference hall better than we found it
+
+If you are interested in volunteering for the conference, please read and fill out the `Call for Volunteers form <https://goo.gl/forms/MprOM1PfOAdD6TKG3>`_. You will receive additional information in early April regarding shifts, detailed duties, and other updates to the volunteer duties and schedules.
+
+Please feel free to email b.yoose@gmail.com if you have any questions about volunteering for the conference.
+
+Tickets are selling out
```

Comment:
This should probably be at the top, as it's the most important thing to communicate I think.